lib/api/core/entrypoints/embedded/index.js

```diff
@@ -80,8 +80,8 @@ class EmbeddedEntryPoint extends EntryPoint {
     }
 
     errorsManager.throw(
-      'entrypoints',
-      'embedded',
+      'network',
+      'embedded_entrypoint',
```

Comment:
(nitpicking)

We distinguish the "embedded"  entrypoint in Kuzzle's source code because we have an entrypoint when a Kuzzle Proxy is used. 
As the proxy is deprecated and will be completely removed in a near-future, there is no longer need to make that distinction.